test/mocks/values/time.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  TIME: '2023-03-30T00:00:00.000Z'
```

Comment:
Ah okay, so can't be changed then in that case?

Reply:
We could rename our mock, but it would be misleading as we'd have a `TIMESTAMP` mock that is a mock for `time` which I don't think we should do.  We can't change the `time` part.

Reply:
Yeah okay, makes sense. That's fine then :)